packages/remix/test/fixtures/07-turborepo/apps/remix-app/remix.config.js

```diff
@@ -1,27 +1,27 @@
-const path = require("node:path");
+const path = require('node:path');
 
-const glob = require("glob");
+const glob = require('glob');
 
 const packages = glob
-  .sync("packages/**/package.json", {
-    cwd: path.join(__dirname, "..", ".."),
-    ignore: ["**/node_modules/**"],
+  .sync('packages/**/package.json', {
+    cwd: path.join(__dirname, '..', '..'),
+    ignore: ['**/node_modules/**'],
     absolute: true,
   })
-  .map((pkg) => path.dirname(pkg));
+  .map(pkg => path.dirname(pkg));
 
 /**
  * @type {import('@remix-run/dev').AppConfig}
  */
 module.exports = {
-  serverBuildTarget: "vercel",
-  appDirectory: "app",
-  ignoredRouteFiles: [".*"],
-  assetsBuildDirectory: "public/build",
+  serverBuildTarget: 'vercel',
+  appDirectory: 'app',
+  ignoredRouteFiles: ['.*'],
+  assetsBuildDirectory: 'public/build',
   // When running locally in development mode, we use the built in remix
   // server. This does not understand the vercel lambda module format,
   // so we default back to the standard build output.
-  server: process.env.NODE_ENV === "development" ? undefined : "./server.js",
+  server: process.env.NODE_ENV === 'development' ? undefined : './server.js',
```

Comment:
Please revert all the formatting changes.

There's another PR that will fix it https://github.com/vercel/vercel/pull/9664